app/Modules/Bookings/BookingsService.php

```diff
@@ -34,9 +34,9 @@ public function validateRatingRequirement(User $user, Position $position): bool
         return ControllerRating::isValidRatingForSuffix($positionSuffix, $ratingValue);
     }
 
-    public function validateBookingTimes(Carbon $from, Carbon $to, Position $position): bool
+    public function validateBookingTimes(Carbon $from, Carbon $to, Position $position, int $excluded = null): bool
     {
-        $bookings = Booking::where('position_id', $position->getKey());
+        $bookings = Booking::where([['position_id', $position->getKey()], ['id', '!=', $excluded]]);
```

Comment:
Might possibly be a bit more readable if these were broken down into two where's - I'll leave that up to you